### ISSUE TITLE
Can't change Range Slider values when slider thumbs are stacked to the left.

### ISSUE THREAD

**Issue by ahrbil** [chakra-ui/chakra-ui#4800]
### Description

When using the Range Slider component I expect to move the thumbs to change range values but if the thumbs are stacked to the left I can't move or drag neither of the thumbs.

### Link to Reproduction

https://codesandbox.io/s/range-slider-7sj2f

### Steps to reproduce

1. Slide all the thumbs to the left so they are stacked.
2. Try to move the thumb.

### Chakra UI Version

1.6.9

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

**Comment by Brennvo** [comment]
Good catch. The issue here is that when the thumbs are stacked on top of each other, only the left thumb is draggable. You can see this by dragging all of the thumbs to the right and then dragging the stack to the left; the left thumb moves.
